lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -18,11 +18,12 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final texts = context.texts();
-    return FutureBuilder<Config>(
-      future: Config.instance(),
-      builder: (BuildContext context, AsyncSnapshot<Config> snapshot) {
+    final mempoolHelper = MempoolHelper();
```

Comment:
I think it is better to resolve the MempoolHelper via the ServiceInjector (where all our services defined).
Also we can rename it to ChainExplorer or similar. The whole idea of abstracting the URL formatting is not to imply on a specific implementation (mempool space).

Reply:
Do we still want to explose the mempoolInstance to the serviceInjetor after we decided to change the scope of the pr from abstracting a chainexplorer to letting users use their own mempool instance?